google-cloud-datastore/src/main/java/com/google/cloud/datastore/Datastore.java

```diff
@@ -168,6 +169,23 @@
    */
   List<Key> allocateId(IncompleteKey... keys);
 
+  /**
+   * Reserve a list of IDs from a complete key
+   *
+   * <p>Example of reserving multiple ids in a single batch.
+   *
+   * <pre>{@code
+   * KeyFactory keyFactory = datastore.newKeyFactory().setKind("MyKind");
+   * Key key1 = keyFactory.newKey(10);
+   * Key key2 = keyFactory.newKey("name");
+   * ReserveIdsResponse response = datastore.reserveIds(key1, key2);
+   *
```

Comment:
this extra line can be deleted

Reply:
yes we can, but i did it same as other javaDoc are there.